frontend/public/components/overview/networking-overview.tsx

```diff
@@ -44,10 +47,11 @@ const ServicesOverviewList: React.SFC<ServiceOverviewListProps> = ({ services })
 
 const RoutesOverviewListItem: React.SFC<RoutesOverviewListItemProps> = ({ route }) => {
   const { name, namespace } = route.metadata;
+  const { t } = useTranslation();
   return (
     <li className="list-group-item">
       <ResourceLink kind="Route" name={name} namespace={namespace} />
-      <span className="text-muted">{'Location: '}</span>
+      <span className="text-muted">{t('public~Location: ')}</span>
```

Comment:
Above you added the space behind the span. Should we use this way here as well. This will reduce the risk that the translation miss that space.

```suggestion
      <span className="text-muted">{t('public~Location:')}</span>{' '}
```